src/main/java/org/carlmontrobotics/commands/AutonRuinerShootAndIntake.java

```diff
@@ -0,0 +1,47 @@
+package org.carlmontrobotics.commands;
+
+import static org.carlmontrobotics.Constants.Armc.SMART_CURRENT_LIMIT_TIMEOUT;
+import static org.carlmontrobotics.Constants.Effectorc.*;
+import org.carlmontrobotics.subsystems.Arm;
+import org.carlmontrobotics.subsystems.IntakeShooter;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class AutonRuinerShootAndIntake extends Command {
+  private IntakeShooter intakeShooter;
+  private Timer timer = new Timer();
+
+  public AutonRuinerShootAndIntake(IntakeShooter intakeShooter) {
+    this.intakeShooter = intakeShooter;
+    addRequirements(intakeShooter);
+    timer.stop();
+    timer.reset();
+  }
+
+  @Override
+  public void initialize() {
+    intakeShooter.setRPMIntake(INTAKE_RPM);
+    intakeShooter.setMaxOuttake(0.5);
```

Comment:
It's generally good practice to avoid just having numbers in there and instead have the number represented by a variable. That way if you need to make a change, you can quickly edit the variable rather than having to go to each individual file and change the numbers. (You use this exact number in `AutonRuinerShoot.java` which I'm assuming should be the same)